monitoring/alertmanager/templates/instances.tmpl

```diff
@@ -0,0 +1,18 @@
+{{ define "slack.instances.title" -}}
```

Comment:
@burnerlee what is the purpose of this file?

Reply:
@Ram81 this is to define the template of the alert message that we are receiving on slack

Reply:
We have 2 templates one for instance-down alerts and one for API Threshold exceeded alerts